packages/vscode-extension/package.json

```diff
@@ -74,6 +74,36 @@
     "onView:teamsfx-help-and-feedback"
   ],
   "contributes": {
+    "icons": {
+      "toolkit-commandBot": {
```

Comment:
What is the scope of these icons? If it's used globally in VS Code, maybe we could name it more specific like `teamsfx-toolkit-command-bot` to prevent conflicts.

Reply:
They can only be used inside Teams Toolkit extension and dependent extensions.